components/src/organisms/FixtureOption/index.tsx

```diff
@@ -36,6 +36,7 @@ export function FixtureOption(props: FixtureOptionProps): JSX.Element {
       <StyledText
         desktopStyle="bodyDefaultSemiBold"
         oddStyle="bodyTextSemiBold"
+        padding={`${SPACING.spacing8}`}
```

Comment:
Do you think it is possible to remove this padding by modifying the above padding size?

```suggestion
        padding={SPACING.spacing8}
```

Reply:
yes good call out! I modified to use the outer padding